recipes/sqlitecpp/all/conanfile.py

```diff
@@ -78,11 +92,15 @@ def generate(self):
         tc.variables["SQLITECPP_BUILD_EXAMPLES"] = False
         tc.variables["SQLITECPP_BUILD_TESTS"] = False
         tc.variables["SQLITECPP_USE_STACK_PROTECTION"] = self.options.stack_protection
+        tc.variables["SQLITE_HAS_CODEC"] = self.options.get_safe("with_sqlcipher", False)
         tc.generate()
 
         tc = CMakeDeps(self)
         tc.generate()
 
+        pc = PkgConfigDeps(self)
```

Comment:
Is this only necessary for the `with_sqlcipher` option set to true? Else, why was this added now and not before? I'd appreciate any insight you might have, thanks! :)

Reply:
You right, I just added a condition